### PR TITLE
Update Sponsors And Speakers

### PR DESCRIPTION
**Pull request by CIPHERTron**
1. Sponsors
- Gold: GitHub, AWS User India, Sashido
- Bronze: Codex, InterviewReady, Invision

2. Speakers
- Update Gaurav Sen Profile

3.  About Section
- Minor Changes

### CODE REVIEW

File: src/screens/LandingScreen/SpeakerSection.js [hunk 1]
```diff
@@ -138,9 +138,9 @@ const SpeakerSection = () => {
                     pb: [1, 3]
                   }}
                 >
-                  Gaurav is a Software Engineer and a Content Creator. He has
-                  YouTube channel on system design and algorithms has more than
-                  300k subscribers.
+                  Gaurav is a Software Engineer and a Content Creator. His
+                  YouTube channel on system design and algorithms boasts more
```
Comment (HarishTeens) [inline src/screens/LandingScreen/SpeakerSection.js:142]: boasts? Are you sure about this sentence?

**Review by HarishTeens — CHANGES_REQUESTED** [review]

**Comment by HarishTeens** [comment]
@CIPHERTron Add the screenshots over here for future reference 🚀

File: src/screens/LandingScreen/SpeakerSection.js (hunk 1 above)
Comment (CIPHERTron) [reply src/screens/LandingScreen/SpeakerSection.js:142]: @HarishTeens It was Vedant's suggestion :P

**Comment by CIPHERTron** [comment]
> @CIPHERTron Add the screenshots over here for future reference 🚀

Added @HarishTeens 👍 

Screenshot (567)
Screenshot (568)
Screenshot (569)

File: src/screens/LandingScreen/SpeakerSection.js (hunk 1 above)
Comment (HarishTeens) [reply src/screens/LandingScreen/SpeakerSection.js:142]: lol okay

**Review by HarishTeens — APPROVED** [review]
All good, ready to merge

**Pull request merged**